reinforcement_learning/rl_deepracer_robomaker_coach_gazebo/common

```diff
@@ -1 +0,0 @@
-../common
```

Comment:
Don't remove this common file as its a symlink to https://github.com/awslabs/amazon-sagemaker-examples/tree/master/reinforcement_learning/common
All the changes in common folder need to be done in the folder above